src/types/drupal/node.ts

```diff
@@ -218,32 +225,17 @@ export interface NodeLandingPage extends DrupalNode {
   field_support_services: NodeSupportService[]
 }
 
-/**
```

Comment:
This and all the comments following were annotations for a TypeDoc implementation (https://typedoc.org/index.html) that was abandoned. Removing these for consistency.